src/xunit.netcore.extensions/SkippedTestCase.cs

```diff
@@ -2,16 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
+using Xunit;
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace Xunit.NetCore.Extensions
 {
     /// <summary>Wraps another test case that should be skipped.</summary>
-    internal sealed class SkippedTestCase : IXunitTestCase
+    internal sealed class SkippedTestCase : LongLivedMarshalByRefObject, IXunitTestCase
```

Comment:
I think that was the goal of creating a custom abstract class that inherits from MarshalByRefObject so that they didn't have to #if-def all of their types and they could just make them all inherit from this one.